frontend/src/components/elements/Metric/styled-components.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled from "@emotion/styled"
+
+export const StyledText = styled.div(({ theme }) => ({
```

Comment:
That is very helpful and totally agree.

Reply:
I'd also do a better name. These styled components are only defined for the Metric component specifically, so it would be nice if you defined something more than StyledText